src/dynamic_honey_badger/builder.rs

```diff
@@ -56,6 +59,19 @@ where
         self
     }
 
+    /// Sets the starting era to the given value.
+    ///
+    /// This can be useful to let a validator that is already part of the consensus rejoin after a
+    /// connection loss. In that case one would query the peers for the current epoch and join at a
+    /// sufficiently later on (to account for network latency). Peers might provide different
```

Comment:
"Peers might provide..." and until the end of the paragraph: This is an interesting idea but maybe let's not mention it at this stage because the design might still change? Let's add it when the implementation is ready.